src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package org.fundacion.jala.converter.view.converter;
+import org.fundacion.jala.converter.view.utilities.JLabelStyle;
+import javax.swing.*;
+import java.awt.*;
+
+class OutputSettingsAudio extends JPanel {
+    private JComboBox volumeSelect;
+    private JComboBox hzSelect;
+    private JComboBox audioChannelSelect;
+    private JCheckBox metadataOption;
+
+    /**
+     * Initialize the graphic elements for output settings configurations.
+     */
+    protected OutputSettingsAudio() {
+        JLabelStyle volumeLabel = new JLabelStyle("Select Volume: ", "h3", 2, 70, 20);
+        JLabelStyle hzLabel = new JLabelStyle("Select Hz: ", "h3", 2, 70, 20);
+        JLabelStyle audioChannelLabel = new JLabelStyle("Select Audio Channel: ", "h3", 2, 70, 20);
+        volumeSelect = new JComboBox();
+        volumeSelect.setFont(new Font("Barlow", 0, 12));
+        volumeSelect.setPreferredSize(new Dimension(70, 30));
+        setVolumeSelect();
+        hzSelect = new JComboBox();
+        hzSelect.setFont(new Font("Barlow", 0, 12));
+        hzSelect.setPreferredSize(new Dimension(70, 30));
+        setHzSelect();
+        audioChannelSelect = new JComboBox();
+        audioChannelSelect.setFont(new Font("Barlow", 0, 12));
+        audioChannelSelect.setPreferredSize(new Dimension(70, 30));
+        setAudioChannelSelect();
+        metadataOption = new JCheckBox("With metadata");
+        metadataOption.setFont(new Font("Barlow", 0, 12));
+        setLayout(new GridLayout(4, 2));
+        add(volumeLabel.getTextLabel());
+
+        add(volumeSelect);
+        add(hzLabel.getTextLabel());
+        add(hzSelect);
+        add(audioChannelLabel.getTextLabel());
+        add(audioChannelSelect);
+        add(metadataOption);
+    }
+
+    /**
+     * Method that set all possible volume
+     * for Audio converter.
+     */
+    protected void setVolumeSelect() {
+        volumeSelect.addItem(" ");
```

Comment:
Don't know if is recommended to set an empty option, maybe can put some element as default?

Reply:
Maybe de default is empty